.github/actions/test/action.yml

```diff
@@ -38,6 +38,7 @@ runs:
       shell: bash
       name: run python tests
     - run: |
+        HOROVOD_WITH_PYTORCH=1 python -m pip install torch==1.12.1 horovod[pytorch] --no-binary horovod
```

Comment:
please, use requirements.txt file instead.

Reply:
I have not been able to get horovod to install properly in the requirements.txt

Reply:
I thought one of the benefits of moving to Ray is that we don't have to deal with such setups anymore.

Reply:
This does not have to do with ray, this is a horovod issue.

Reply:
I though ray supposed to take care of underlying training infrastructure. Why do we still need horovod?

Reply:
Ray manages horovod for us but we still need to install it.

Reply:
Does ray use a different trainer for horovod?

Reply:
Yes, it has HorovodTrainer which uses horovod as a backend, see GAT model